releasenotes/notes/extend-backend-defaults-4370f983d599a26b.yaml

```diff
@@ -0,0 +1,27 @@
+---
+features:
+  - |
+    PulseDefaults (accessed normally as backend.defaults()) has an attribute,
+    `instruction_schedules` which has the methods of CmdDef. CmdDef will be
+    deprecated. The new `instruction_schedules` is an `InstructionScheduleMap`
+    object with three new functions beyond what CmdDef had:
+      - qubit_instructions(qubits) returns the operations defined for the qubits
+      - assert_has(instruction, qubits) raises an error if the op isn't defined
+      - remove(instruction, qubits) like pop, but doesn't require parameters
+    There are some differences from the CmdDef:
+      - __init__ takes no arguments
+      - cmds and cmd_qubits are deprecated and replaced with instructions and
+        qubits_with_instruction
+
+    Example:
+        backend = provider.get_backend(backend_name)
+        inst_map = backend.defaults().instruction_schedules
+        qubit = inst_map.qubits_with_instruction('u3')[0]
+        x_gate = inst_map.get('u3', qubit, P0=np.pi, P1=0, P2=np.pi)
+        pulse_schedule = x_gate(DriveChannel(qubit))
+
+other:
```

Comment:
This should probably be an upgrade note not other because it's something people we need to adapt to when upgrading from a previous version.